rusoto/core/src/region.rs

```diff
@@ -28,6 +28,7 @@ pub enum Region {
     UsWest1,
     UsWest2,
     CnNorth1,
+    Local(String)
```

Comment:
Since this new Region type can be specified as anything, perhaps `Unknown` or `Custom` would be more accurate.